src/main/java/org/profit/telegrambot/ButtonService.java

```diff
@@ -0,0 +1,60 @@
+package org.example;
+
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardButton;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class ButtonService {
+
+    public ReplyKeyboardMarkup setButtons(List<KeyboardRow> keyboardRowList) {
+        ReplyKeyboardMarkup replyKeyboardMarkup = new ReplyKeyboardMarkup();
+        replyKeyboardMarkup.setSelective(true);
+        replyKeyboardMarkup.setResizeKeyboard(true);
+        replyKeyboardMarkup.setOneTimeKeyboard(true);
+        replyKeyboardMarkup.setKeyboard(keyboardRowList);
+        return replyKeyboardMarkup;
+    }
+
+    public List<KeyboardRow> createButtons(List<String> buttonsName) {
+        List<KeyboardRow> keyboardRows = new ArrayList();
+        KeyboardRow keyboardRow = new KeyboardRow();
+        Iterator var4 = buttonsName.iterator();
+
+        while(var4.hasNext()) {
```

Comment:
Could you please give a more informative name to the variable?
At the first glance, it is difficult for me to understand what `var4` means.